docs/5.0/preview/upcoming-releases.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Teleport Upcoming Releases
+description: A timeline of teleport upcoming releases.
+---
+
+# Upcoming Teleport Releases
+
+The teleport team delivers a new major release every 3 months.
+
+## Teleport 6.0 "San Diego"
+
+The team open sources role based access controls, implements Database Access, Session Termination
+and Access Workflows UI.
+
+
+### Release Schedule
+
+|Version              | Date              | Description
+|---------------------|-------------------|---------------------------
+|First alpha          | Jan 15th, 2021    | Good for testing and demos.
+|First beta           | Feb 1st, 2021     | Deploy on staging.
+|Release              | March 1st, 2021   | Good to go for production.
```

Comment:
For Github Milestones, should the "Milestone" date be this one, or the first beta date?

Reply:
the release one, people usually care more about it.